src/main/java/org/shanerx/tradeshop/util/JsonConfiguration.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *     Copyright (c) 2016-2017 SparklingComet @ http://shanerx.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * NOTICE: All modifications made by others to the source code belong
+ * to the respective contributor. No contributor should be held liable for
+ * any damages of any kind, whether be material or moral, which were
+ * caused by their contribution(s) to the project. See the full License for more information
+ */
+
+package org.shanerx.tradeshop.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
+public class JsonConfiguration
```

Comment:
Now that I see this, I remember an old project I wrote - that does exactly this... silly me that I didn’t remember this. It has a couple utility methods we could copy over.
([Click](https://github.com/SparklingComet/ConfigAPI)

Reply:
This class should implement the `Serializable` Java standard API interface.